multiaccounts/accounts/database.go

```diff
@@ -454,7 +465,10 @@ func (db *Database) GetSettings() (Settings, error) {
 		&s.WalletSetUpPassed,
 		&s.WalletVisibleTokens,
 		&s.WakuBloomFilterMode,
-		&s.WebViewAllowPermissionRequests)
+		&s.WebViewAllowPermissionRequests,
+		&sqlite.JSONBlob{Data: &s.CurrentUserStatus},
```

Comment:
I'd avoid using a blob, as it's difficult to migrate in case of schema changes (although this looks like a fairly safe usage), probably I'd have either a separate table, or flatten the field in settings, though if we are confident it won't cause issues in the future we are probably ok, up to you